dataproc_jupyter_plugin/services/airflow.py

```diff
@@ -66,10 +70,12 @@ async def list_jobs(self, composer_name):
         airflow_uri, bucket = await self.get_airflow_uri(composer_name)
         try:
             api_endpoint = f"{airflow_uri}/api/v1/dags?tags={TAGS}"
-            response = requests.get(api_endpoint, headers=self.create_headers())
-            if response.status_code == 200:
-                resp = response.json()
-            return resp, bucket
+            async with self.client_session.get(
+                api_endpoint, headers=self.create_headers()
+            ) as response:
+                if response.status == 200:
```

Comment:
Report the response back to the user when it isn't a 200, so that they can debug and fix whatever issue they encountered.